plugins/lua/stockpiles.lua

```diff
@@ -508,6 +508,16 @@ function StockpilesOverlay:init()
                                 {label='Training', value=false}},
                         initial_option=false,
                         on_change=self:callback('toggleLogisticsFeature', 'train'),
+                    }, widgets.CycleHotkeyLabel{
+                        view_id='forbid',
+                        frame={t=2, l=0, w=16},
```

Comment:
can you put some other toggles down on `t=2` and make the overlay panel a bit shorter?

Reply:
Yeah, I was thinking that would be possible - I had some crazier ideas too but for now I'll see if I can get it looking good with options in distributed in two rows.